clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1205,6 +1248,14 @@ public void assign(Collection<TopicPartition> partitions) {
                 throw new IllegalStateException("Consumer is not subscribed to any topics or assigned any partitions");
             }
 
+            // TODO: Record poll time in this method?
```

Comment:
@harshach @omkreddy @hachikuji @bbejeck Is this where we should be recording poll times?